oq_consolidate.py

```diff
@@ -31,14 +31,14 @@
 
 from qgis.core import Qgis
 
-from . import qconsolidatedialog
-from . import aboutdialog
+from . import oq_consolidate_dialog
+from . import about_dialog
 from .utils import log_msg, tr
 
 from . import resources_rc  # NOQA
 
 
-class QConsolidatePlugin(object):
+class OQConsolidatePlugin(object):
```

Comment:
In general we should consider making this plugin QConsolidate and then add on top of it add the little specificity that we have ad OQConsolidate